docusaurus.config.js

```diff
@@ -19,7 +19,7 @@ const config = {
 	// Set the /<baseUrl>/ pathname under which your site is served
 	// For GitHub pages deployment, it is often '/<projectName>/'
 	baseUrl: "/",
-	trailingSlash: true,
+	trailingSlash: false,
```

Comment:
Will this break links in the existing userbase? Eg: we have doc links with a `/` distributed in desktop apps, and those links may end up unreachable?
https://github.com/phcode-dev/phoenix/blob/main/src/assets/default-project/en/Newly_added_features.md?plain=1

Reply:
Yes, you're right. This change might break those links.